### ISSUE TITLE
Nextjs refactor: Centering button behavior when viewing a location's summary (mobile) or details (desktop)

### ISSUE THREAD

**Issue by adambard1** [streetlives/yourpeer.nyc#177]
- [ ] If I have been scrolling around and select a pin to view it's details, then click the center button, the selected location should zoom out and remain in view on the map window as centering shifts the map to my location., so I can get a sense of where it is in relation to me.

This matches behaviour on prod.

**Comment by jbeard4** [comment]
Fixed 
![Screenshot from 2024-07-23 23-12-06](https://github.com/user-attachments/assets/f00bffd5-5119-435f-9782-eceff533baff)
![Screenshot from 2024-07-23 23-11-29](https://github.com/user-attachments/assets/e81c03fb-44c1-41d1-a648-d76c021e3ff8)

**Comment by adambard1** [comment]
I'm not sure if this is fixed for local NYC views, but given your screenshot, I presume so. However, I don't think we want to show the user's location if they are outside the 26-mile boundary from the center of Central Park—we would probably use the default location at the center of Central Park for this if they're not in NYC.
Also (not that there's any reason to do this, I would apply the above suggestion), I'm in the UK, and it doesn't show both locations.


![Image](https://github.com/user-attachments/assets/22450ddd-51f0-4610-a7e2-c73d15c4ba37)

**Comment by jbeard4** [comment]
@adambard1 The issue you mentioned has been fixed - it will default to central park if the user's current location is out of range.

**Comment by adambard1** [comment]
Yes, works well imo, though perhaps we'll want to have a popup at some point explaining that the user needs to be inside the NYC metro area and to turn on their location to enable centering on their location.
